pyproject.toml

```diff
@@ -17,3 +36,27 @@ exclude_lines = [
 
 [tool.pytest.ini_options]
 pythonpath = ["postcode_lookup"]
+
+[tool.uv.sources]
```

Comment:
This is a bit of a nitpick, but our pyproject.toml has 2 categories of thing in it:

- Package management stuff (`project`, `dependency-groups`, `tool.uv.<things>`)
- Config for unrelated tools that happen to live in pyproject.toml (`tool.ruff`, `tool.pytest`)

I'd like us to group all the package management stuff together in the file seperately from the unrelated config just for readability. At the moment it is a bit of a random mashup.

Reply:
https://github.com/DemocracyClub/ec-postcode-lookup-pages/pull/117/commits/2a83546288f2735f52d7537aff862eac2284eb62